databases/week2/part1-and-part3.sql

```diff
@@ -0,0 +1,73 @@
+/*part1: Working with Tasks*/
+SELECT * 
+FROM task;
+
+SELECT * 
+FROM user;
+
+SELECT * 
+FROM status;
+
+SELECT * 
+FROM user_task;
+
+/*Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id */
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES ('Teaching', 'Telling somthing', '2018-10-25 07:54:16', '2019-10-15 13:05:09', null, 2);
+
+SELECT * 
+FROM task;
+
+INSERT INTO user_task (user_id, task_id)
+VALUES (4,13);
+
+INSERT INTO user_task (user_id, task_id)
+VALUES (2,36);
+
+SELECT * 
+FROM user_task;
+
+/*Change the title of a task*/
+UPDATE task
+SET title='Teacher',due_date = '2021-11-23 10:13:42', status_id =3
+WHERE ID=36;
+
+DELETE
+FROM task
+WHERE ID=35;
+
+SELECT * 
+FROM task;
+
+/*Part3 :More queries*/
+/*Get all the tasks assigned to users whose email ends in @spotify.com*/
+SELECT * 
+FROM user
+WHERE email LIKE '%@spotify.com';
+
+/*Get all the tasks for 'Donald Duck' with status 'Not started'*/
+
+SELECT * 
```

Comment:
nice job :), you can try it with join as well

Reply:
Thankyou for your comments.I have updated according to your comments. :)

Reply:
Please let me know @aina21...if I can merge now?